config/validator/validation.yaml

```diff
@@ -50,18 +50,14 @@ App\User\Application\DTO\UserPatchDto:
     initials:
       - Length:
           max: 255
-      - App\Shared\Application\Validator\Initials:
-          optional: true
+      - App\Shared\Application\Validator\Initials: ~
```

Comment:
Initials validator has a bug when you put name and surname with space between them, then we got an validation error, but we should allow any initials except empty strings, starting with numbers or only spaces
Please check best practices for initials validation for any variants of human names, surnames and so on